docs/StardustDocs/topics/gettingStarted/Modules.md

```diff
@@ -0,0 +1,163 @@
+# Modules
+
+Kotlin DataFrame is split into specific modules that give you the flexibility
+to only use the modules you need.
+In this topic you'll learn what these modules are and how to add module dependencies
+to an existing Gradle/Maven project.
+
+## Configure the repository
+
+Kotlin DataFrame modules are available from the Maven Central repository.
+To use them, add the appropriate dependency into your repositories mapping:
+
+<tabs>
+  <tab title="Kotlin DSL"> 
+
+```kotlin
+repositories {
+    mavenCentral()
+}
+```
+
+</tab>
+  <tab title="Groovy DSL">
+
+```kotlin
+repositories {
+    mavenCentral()
+}
+```
+
+</tab>
+</tabs>
+
+## General Kotlin DataFrame dependency
+
+If you don't need custom module adjustments, you can easily add a general dependency with  
+all [core Kotlin DataFrame modules](#core-kotlin-dataframe-modules) — `dataframe-core` and all officially 
+supported IO format modules, excluding [experimental ones](#experimental-kotlindataframe-modules).
+
+<tabs>
+<tab title="Kotlin DSL">
+
+```kotlin
+dependencies {
+    implementation("org.jetbrains.kotlinx:dataframe:%dataFrameVersion%")
+}
+```
+
+</tab>
+
+<tab title="Groovy DSL">
+
+```groovy
+dependencies {
+    implementation 'org.jetbrains.kotlinx:dataframe:%dataFrameVersion%'
+}
+```
+
+</tab>
+</tabs>
+
+## Core Kotlin DataFrame modules
+
+Core Kotlin DataFrame modules are `dataframe-core`, which contains all logic 
+related to working with DataFrame,  
+and optional modules for supporting the most important IO types:
+
+| Module            | Function                                                                                  |
+|-------------------|-------------------------------------------------------------------------------------------|
+| `dataframe-core`  | The [DataFrame](DataFrame.md) API and its implementation.                                 |
+| `dataframe-json`  | Provides support for JSON format writing and reading.                                     |
+| `dataframe-csv`   | Provides support for CSV format writing and reading.                                      |
+| `dataframe-excel` | Provides support for XSL/XLSX format writing and reading.                                 |
+| `dataframe-jdbc`  | Provides support for JDBC data sources writing and reading.                               |
```

Comment:
For now only reading, right?

Reply:
Oh, really? I haven't known xD.

Reply:
We have a ticket, but not supporting writing